src/software/ai/hl/stp/action/all_actions.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "software/ai/hl/stp/actions/chip_action.h"
```

Comment:
can this file just be the output of a genrule so we have less things to worry about when adding a new action?

Reply:
I think @jonathanlew brought that up separately. @jonathanlew is there a ticket for this? (I didn't see one)

I believe we would address this separately (would need the same for tactics, primitives, and really anything with a visitor), but that can be @garethellis0 's call

Reply:
Yep, #865 is open for this, seems out-of-scope for this PR.